.eslintrc.js

```diff
@@ -29,8 +29,5 @@ module.exports = {
   rules: {
     "import/order": "off",
     "react-hooks/exhaustive-deps": "error",
-    "prettier/prettier": ["warn", {
```

Comment:
why was this removed? this will require a change in indentation for all the files in the code base right? 😰